sos/plugins/krb5.py

```diff
@@ -19,8 +19,34 @@ class Krb5(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin):
     packages = ('krb5-libs', 'krb5-user')
 
     def setup(self):
-        self.add_copy_spec("/etc/krb5.conf")
+        self.add_copy_spec([
+            "/etc/krb5.conf",
+            "/etc/krb5.conf.d/*",
+            "/var/log/krb5kdc.log",
+            "/var/log/kadmind.log",
```

Comment:
So, syntactically the trailing comma on multiline lists is okay, but stylistically we don't do this anywhere else in sos (at least not that I'm aware of).

I'm inclined to say for consistency sake, let's remove the trailing commas. @bmr-cymru thoughts?

Reply:
There are odd examples of it. I don't have a huge problem either way (to be honest, I agree with the argument that it reduces the chance of continuation/concatenation bugs, but it just looks so ... ugly :-/ ).

Reply:
I'd love something that could semantically validate lists and warn on missing commas - we have some cases of genuine concatenation but they are rare and easy to filter.